libraries/apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ApolloResponse.kt

```diff
@@ -128,6 +128,16 @@ private constructor(
         .isLast(isLast)
   }
 
+  override fun toString(): String {
+    return "ApolloResponse(" +
+        "operationName=${operation.name()}, " +
+        "data=${if (data == null) "null" else "${operation.name()}.Data"}, " +
+        "errors=${errors?.size ?: "null"}, " +
+        "exception=${if (exception == null) "null" else exception::class.simpleName ?: "true"}, " +
+        "isLast=$isLast" +
```

Comment:
Can you remove `isLast` here? I'm not happy with this field and will probably deprecate it in the mid term. I don't want to advertise it too much.